server/routes/prisonIncentiveLevels.ts

```diff
@@ -93,11 +93,23 @@ export default function routes(router: Router): Router {
         const incentivesApi = new IncentivesApi(res.locals.user.token)
         const { levelCode } = req.params
         const { id: prisonId, name: prisonName } = res.locals.user.activeCaseload
+
+        const [incentiveLevel, prisonIncentiveLevel] = await Promise.all([
+          incentivesApi.getIncentiveLevel(levelCode),
+          incentivesApi.getPrisonIncentiveLevel(prisonId, levelCode),
+        ])
+        if (!prisonIncentiveLevel.active) {
+          throw NotFound('Prison incentive level is already inactive')
+        }
+        if (incentiveLevel.required) {
+          throw NotFound('Incentive level is required globally')
+        }
```

Comment:
See comment above related to this. A `400` may be more appropriate.

Reply:
done! 26fc4ada68ad421a8b5e25a3587a23a0ba683ce5